conf/modules/allele_specific_calling.config

```diff
@@ -49,11 +49,30 @@ process {
         ]
     }
 
+    withName: '.*ALLELE_SPECIFIC_CALLING:SPLIT_MULTIALLELICS' {
+        ext.prefix = { "${meta.id}_norm" }
+        ext.args = '--output-type z --multiallelics -both'
+    }
+
     withName: '.*ALLELE_SPECIFIC_CALLING:TABIX_TABIX' {
         publishDir = [
             path: { "${params.outdir}/bootstrapann" },
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
```

Comment:
remove this publishdir and add it to `TABIX_ANNOTATE`

Reply:
Done